### DISCUSSION TITLE
Auto synchronization of the folder on the cloud

### DISCUSSION THREAD

**Discussion by bottarocarlo** [abraunegg/onedrive#2361] [Q&A] [answered]
Hi every one, just a simple question, how can I start monitoring file changes in the folder and automatic upload it on one drive?

**Comment by abraunegg** [comment, marked as answer]
@bottarocarlo 

Please read the documentation: https://github.com/abraunegg/onedrive/tree/master/docs

Start with: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md
